src/Twig/Extension/SonataTemplatesExtension.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\AdminBundle\Twig\Extension;
+
+use Sonata\AdminBundle\Admin\AdminInterface;
+use Sonata\AdminBundle\Admin\Pool;
+use Symfony\Component\HttpFoundation\RequestStack;
+use Twig\Extension\AbstractExtension;
+use Twig\TwigFunction;
+
+class SonataTemplatesExtension extends AbstractExtension
```

Comment:
Should be `final`

Reply:
Also, please avoid the plural, find something like `TemplateList` or `TemplateRegistry`. Why the `Sonata` prefix, BTW?

Reply:
I'll change it. I took the Sonata prefix from the existing Twig extension.

Reply:
Ah right. Please remove it anyway, it's already in the namespace